src/components/collection/DataPreview/index.tsx

```diff
@@ -46,7 +52,7 @@ export function DataPreview({ collectionName }: Props) {
     const isLoading = journalsLoading || journalData.loading;
 
     return (
-        <>
+        <Grid item xs={12} sx={{ m: 2 }}>
```

Comment:
Initially I was going to try switching to a grid layout where things can feel more responsive. However, didn't wanna rock the boat. So leaving it in as we'll probably need some grid support later.

This goes for all the grid stuff related to details.